gammapy/makers/reduce.py

```diff
@@ -102,7 +102,10 @@ def make_dataset(self, dataset, observation):
         log.info(f"Computing dataset for observation {observation.obs_id}")
         for maker in self.makers:
             log.info(f"Running {maker.tag}")
-            dataset_obs = maker.run(dataset=dataset_obs, observation=observation)
+            try:
```

Comment:
Why this try clause?

Reply:
RingBackground and AdaptiveRingBackground don't have observation in their arguemnt contrary to any other makers

Reply:
Actually for consistency that could just take `.run(dataset, observation=None)`, maybe it is needed later for access to meta data or similar...

Reply:
done